packages/orchestrator/lib/routes/v1/task/taskId/output.ts

```diff
@@ -33,13 +33,10 @@ const getHandler = (scheduler: Scheduler) => {
             case 'STARTED':
                 return res.status(204).send(); // No content yet
             case 'SUCCEEDED':
-                return res.status(200).json({ output: task.value.output });
             case 'FAILED':
-                return res.status(404).json({ error: { code: 'task_failed', message: `failed` } });
             case 'EXPIRED':
-                return res.status(404).json({ error: { code: 'task_expired', message: `expired` } });
             case 'CANCELLED':
-                return res.status(404).json({ error: { code: 'task_cancelled', message: `cancelled` } });
+                return res.status(200).json({ state: task.value.state, output: task.value.output });
```

Comment:
`FAILED` and `EXPIRED` respond with a 200?

Reply:
yes, since this is the `/output` endpoint and there is potentially an output it is now a 200